src/index/updater/inscription_updater.rs

```diff
@@ -201,13 +203,14 @@ impl<'a, 'db, 'tx> InscriptionUpdater<'a, 'db, 'tx> {
           inscription_id,
           offset,
           origin: Origin::New {
-            reinscription: inscribed_offsets.get(&offset).is_some(),
-            cursed: curse.is_some(),
+            cursed: curse.is_some() && !jubilant,
             fee: 0,
```

Comment:
Re-inscribing on a vindicated inscription will continue to yield a vindicated inscription, which may inconsistent with the process before Jubilee.

```
            .inscription_number
              < 0;
            if initial_inscription_is_cursed {
```

Reply:
I don't think I understand.

Can you elaborate? The specific sequence of transactions that would cause a problem would be helpful.

Reply:
Before Jubilee, first inscribe a cursed inscription using Pushnum. Continuing to inscribe on the same sat will result in a blessed inscription.

After Jubilee, first inscribe a Vindicate inscription using Pushnum, and continuing to inscribe on the same sat will result in a Vindicate inscription, not a blessed one.

Reply:
We can fix that by also checking the vindicated charm. `initial_inscription_is_cursed` should be true if either inscription_number < 0 or inscription has vindicated charm.

Reply:
> We can fix that by also checking the vindicated charm. `initial_inscription_is_cursed` should be true if either inscription_number < 0 or inscription has vindicated charm.
yes, we also think this is what we want, and `vindicated` will be consistant with release 0.9 after being fixed
```Rust
let entry = InscriptionEntry::load(
  self
    .sequence_number_to_entry
    .get(initial_inscription_sequence_number)?
    .unwrap()
    .value(),
);

let initial_inscription_is_cursed =
  entry.inscription_number < 0 || Charm::Vindicated.is_set(entry.charms);
```

Reply:
Nice, thanks for the explanation! This is now fixed.